browser/brave_wallet/brave_wallet_provider_impl_unittest.cc

```diff
@@ -116,6 +111,13 @@ class BraveWalletProviderImplUnitTest : public testing::Test {
   HostContentSettingsMap* host_content_settings_map() {
     return HostContentSettingsMapFactory::GetForProfile(&profile_);
   }
+  void CreateWalletAndAccount() {
+    keyring_controller()->CreateWallet(
+        "testing123", base::DoNothing::Once<const std::string&>());
+    base::RunLoop().RunUntilIdle();
+    keyring_controller()->AddAccount("Account 1",
+                                     base::DoNothing::Once<bool>());
```

Comment:
This also need another `base::RunLoop().RunUntilIdle();`

Reply:
Fixed and rebased in a9ead3b050b5e750c0bb0f182c0590a8182ff96e